src/components/atoms/Spinner/Spinner.tsx

```diff
@@ -5,6 +5,7 @@ import actionSpinner from './spinner-action.gif';
 import deprecated from 'deprecated-prop-type';
 import { useThemeContext } from '../../styles/Theme';
 import jlSpinner from '../../../content/images/jl-spinner.gif';
```

Comment:
I guess we shouldn't reference `jl` specifically in the open source lib, maybe we should rename this to _something_

Reply:
shall we call it 'nonBrandedSpinner' ?

Reply:
since its just black and white it would make sense to refer to it as nonBranded

Reply:
Bear in mind we implemented it like this though ` const animationData = theme?.spinner?.spinnerTheme === 'jl'`. Should we change this too?

Reply:
We could do `theme?.spinner?.spinnerTheme === 'unBranded`?